### PR TITLE
[1LP][RFR] Automate: test_import_report_preserve_owner

### PR DESCRIPTION
**Pull request by valaparthvi**
## Purpose or Intent

- __Adding tests__ 
    1. `test_import_report_preserve_owner`

- __Enhancements__
    1. Apply black and remove unused arguments.
    2. Convert all formattings to use f-string

### PRT Run

{{ pytest: cfme/tests/intelligence/reports/test_reports.py -k "test_import_report_preserve_owner" -vvv }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit ae1d72a21538201bf230a85f85d5cb0d1247e3d7

The *local* fixture **`create_custom_tag`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**
    - *test_reports_custom_tags*

The *local* fixture **`rbac_api`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**
    - *test_non_admin_user_reports_access_rest*

The *local* fixture **`create_po_user_and_group`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/intelligence/reports/test_reports.py [hunk 1]
```diff
@@ -170,11 +166,23 @@ def _check_file_size(file_path, expected_size):
...
+@pytest.fixture
+def create_po_user_and_group(request, appliance):
+    """This fixture creates custom user with tenant attached"""
+    group = _groups(
+        request,
+        appliance,
+        appliance.rest_api.collections.roles.get(name="EvmRole-super_administrator"),
+        description="Preserve Owner Report Group",
+    )
+    _users(request, appliance, group=group.description, userid="pouser")
```
Comment (kedark3) [inline cfme/tests/intelligence/reports/test_reports.py]: **Question** : Do we want to cleanup these user/groups afterwards?

File: cfme/tests/intelligence/reports/test_reports.py [hunk 2]
```diff
@@ -116,9 +117,7 @@ def create_custom_tag(appliance):
     tag = appliance.rest_api.collections.tags.action.create(
         {
             "name": "{cat_name}_entry".format(cat_name=category_name),
-            "description": "{cat_name}_entry_description".format(
-                cat_name=category_name
-            ),
+            "description": "{cat_name}_entry_description".format(cat_name=category_name),
```
Comment (sshveta) [inline cfme/tests/intelligence/reports/test_reports.py]: While you are at it ,let's use f string

File: cfme/tests/intelligence/reports/test_reports.py (hunk 1 above)
Comment (valaparthvi) [reply cfme/tests/intelligence/reports/test_reports.py]: Nope, that will be done by the `cfme.rest.gen_data._creating_skeleton` function which is used by `_users`.

File: cfme/tests/intelligence/reports/test_reports.py (hunk 2 above)
Comment (valaparthvi) [reply cfme/tests/intelligence/reports/test_reports.py]: Done! And modified all the other strings as well.

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 08009cc1ffb19594b5d718f06a556b4c19bfc0bb

The *local* fixture **`set_and_get_tenant_quota`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**
    - *test_queue_tenant_quota_reports*
    - *test_report_fullscreen_enabled*
    - *test_reports_filter_content*
    - *test_reports_sort_column*

The *local* fixture **`create_custom_tag`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**
    - *test_reports_custom_tags*

The *local* fixture **`rbac_api`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**
    - *test_non_admin_user_reports_access_rest*

The *local* fixture **`create_po_user_and_group`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**
    - *test_import_report_preserve_owner*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**